console/console-init/ui/src/utils/utils.ts

```diff
@@ -280,7 +280,11 @@ const getLabelByKey = (key: string) => {
     "pwd-hash": "Password",
     hashed_password: "Password",
     psk: "PSK",
-    "x-509": "X-509 certificate"
+    "x-509": "X-509 certificate",
+    "subject-dn": "Subject-dn",
+    "public-key": "Public key",
+    "auto-provisioning-enabled": "Auto-provision",
+    algorithm: "Algorithm"
```

Comment:
I think we don't need this kind of key label (algorithm: "Algorithm") for first letter cap. You can handle by using css textTransform: "capitalize"